spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/AbstractRabbitListenerContainerFactory.java

```diff
@@ -176,6 +176,14 @@ public void setDefaultRequeueRejected(Boolean requeueRejected) {
 	}
 
 	/**
+	 * @return the advice chain that was set. Defaults to {@code null}.
+	 * @since 1.7.4
+	 */
+	public Advice[] getAdviceChain() {
+		return adviceChain;
```

Comment:
Adding an adder with an array as the backing field is kinda awkward, since you need to allocate a new array. The whole factory API simply mirrors the methods from the listener container everywhere and doesn't try to add any convenience methods anywhere else. Also, my use case is a rather exotic one; it isn't even obvious from the documentation *what* you're advising with the advice chain, you have to check the source code for that, and even then you only need this when you have multiple methods that want to add advice (like with Boot), so not many people will need this. Therefore I felt that going beyond adding a simple getter might be overkill. 

What exactly are you missing in the commit message?